marlowe-runtime/cip30-demo/Main.hs

```diff
@@ -21,6 +32,28 @@ getHomeR :: Handler Yesod.Html
 getHomeR = Yesod.defaultLayout do
   Yesod.toWidget $(juliusFile "./cip30-demo/main.js")
 
+getWalletR :: WalletName -> Handler Yesod.Html
+getWalletR walletName = Yesod.defaultLayout do
+  Yesod.toWidget [Yesod.hamlet|
+    <h1>#{walletName} wallet options
+    <ul>
+      <li>
+        <h2><a>create
+        <i>Create a new contract
+      <li>
+        <h2><a>deposit
+        <i>Deposit funds into running contract
+      <li>
+        <h2><a>choose
+        <i>Make a choice in a running contract
+      <li>
+        <h2><a>notify
+        <i>Poke a running contract to continue
+      <li>
+        <h2><a>withdraw
+        <i>Withdraw payouts from a running contract
```

Comment:
@jhbertra I wish to draw your attention to these descriptions. Did I get it right?